src/theme/material/templates/edit-cell-date.tpl.html

```diff
@@ -14,7 +14,7 @@
                 <mat-datepicker-toggle matSuffix
                                        [for]="picker">
                 </mat-datepicker-toggle>
-                <mat-datepicker #picker>
+                <mat-datepicker #picker [panelClass]="$view.edit.cell.theme">
```

Comment:
add a special service theme and call it instead of edit so it would be `$view.theme.class`